planning/scenario_planning/lane_driving/behavior_planning/lane_change_planner/src/utilities.cpp

```diff
@@ -415,13 +420,13 @@ double getDistance3d(const geometry_msgs::msg::Point & p1, const geometry_msgs::
 
 double getDistanceBetweenPredictedPaths(
   const PredictedPath & object_path, const PredictedPath & ego_path, const double start_time,
-  const double end_time, const double resolution, const rclcpp::Logger & logger,
-  const rclcpp::Clock::SharedPtr & clock)
+  const double end_time, const double resolution,
+  const rclcpp::Logger & logger, const rclcpp::Clock::SharedPtr & clock)
 {
   rclcpp::Duration t_delta(resolution);
   double min_distance = std::numeric_limits<double>::max();
-  rclcpp::Time ros_start_time = clock->now() + rclcpp::Duration::from_seconds(start_time);
-  rclcpp::Time ros_end_time = clock->now() + rclcpp::Duration::from_seconds(end_time);
+  rclcpp::Time ros_start_time = clock->now() + rclcpp::Duration(start_time);
```

Comment:
I guess it's better to use from_seconds function.

Reply:
Fixed.